net/acme/Makefile

```diff
@@ -8,17 +8,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=acme
-PKG_SOURCE_VERSION:=db3264ab8c8ee1aee1a84407702db4375f67765b
-PKG_VERSION:=1.6
-PKG_RELEASE:=4
+PKG_VERSION:=2.77
```

Comment:
This is not actually the version numbering scheme used by upstream, though. Should be 2.7.8 (and commit hash updated accordingly).

Reply:
typo. will fix.